tests/manage/test_create_pvc_random_storage_class.py

```diff
@@ -93,10 +93,12 @@ def create_pvc(storageclass_list, count=1):
         storageclass_list (list): This will contain storageclass list
         count (int): count specify no of pvc want's to create
     """
+    global PVC_OBJS
```

Comment:
We would like to avoid using globals

Reply:
any chance to fix this in future PR as this blocks all deployments?

Reply:
Sure, can be resolved other time.   There is other PR #405 from me where I don't use this globals and changing more fixtures in this PR than this is doing. So depends on the approach we like.

I am approving this one, just look at other PR WDYT about it I also added it to @andrewschoen issue about fixtures for discussion if this is the approach we would like to use or not.